pkg/application/app_test.go

```diff
@@ -119,7 +119,7 @@ components:
 			raw:           yaml3,
 			ExpName:       "",
 			InValid:       true,
-			InvalidReason: errors.New("name is required"),
+			InvalidReason: errors.New("please provide an existed App name"),
```

Comment:
What does this mean?

Reply:
For details, please refer to https://github.com/oam-dev/kubevela/issues/320:)